docs/source/installation.md

```diff
@@ -168,13 +168,13 @@ Here is an example of how this can be used on a filesystem that is shared betwee
 On the instance with the normal network run your program which will download and cache models (and optionally datasets if you use 🤗 Datasets). For example:
 
 ```
-python examples/seq2seq/run_seq2seq.py --model_name_or_path t5-small --dataset_name wmt16 --dataset_config ro-en ...
```

Comment:
I'm also very much in favor of `run_translation.py` and `run_summarization.py` - I don't see any benefit in shortening the name here. `trans` is a less known abbreviation than `clm` or `mlm` in NLP IMO. I also don't like `run_sum` , `sum` is often used for summarization. 

I'd rather change `mlm` to `masked_language_modeling` than use `run_sum` here